src/data/database.js

```diff
@@ -427,6 +434,7 @@ class Database {
      */
   async createPayeeQuoteParty (txn, quoteId, party, amount, currency) {
     // note amount is negative for payee and positive for payer
+    // TODO: enums needed
```

Comment:
Would you add there to an enum and use it in the function call as it it used in many places

Reply:
please update these with enums